R/dl-cur-hosp-data.R

```diff
@@ -41,69 +41,60 @@ current_hosp_data <- function() {
 
     # URL for file
     url <- "https://data.cms.gov/provider-data/sites/default/files/archive/Hospitals/current/hospitals_current_data.zip"
-    f_name <- basename(url)
 
-    # Create a temporary file to store the zip file
-    tmp <- tempfile()
+    # Create a temporary directory to process the zip file
     tmp_dir <- tempdir()
+    download_location <- file.path(tmp_dir, "download.zip")
+    extract_location <- file.path(tmp_dir, "extract")
```

Comment:
I didn't see a relationship between `tmp` and `tmp_dir` and the final step was 
`unlink(tmp, recursive = TRUE)` which I think would leave `tmp_dir` in the users temp files.
The change I made here was to use `tmp_dir` as the base and then use 
`unlink(tmp_dir, recursive = TRUE)` at the end